.circleci/config.yml

```diff
@@ -347,6 +347,71 @@ jobs:
             conda activate python${PYTHON_VERSION}
             python -c "import torchtext"
 
+  cachesetup_linux:
+    <<: *binary_common
+    docker:
+      - image: "pytorch/manylinux-cuda102"
+    resource_class: 2xlarge+
+    steps:
+      - checkout
+      - designate_upload_channel
+      - load_conda_channel_flags
+      - run:
+          name: Generate cache key
+          # This will refresh cache on Sundays, nightly build should generate new cache.
+          command: |
+            echo "$(date +"Year-Week %Y-%U")" > .circleci-weekly
+            echo "$(date +"Year-Date %Y-%d")" > .circleci-daily
+      - restore_cache:
+          keys:
+
+            - data-linux-v1-{{ checksum ".circleci-daily" }}
+
+      - run:
+          name: Exit if cache exists
+          command: |
+            FILE=.data/cache_status_file.json
+            if test -f "$FILE"; then
+              cat $FILE
+              circleci step halt
+            else
+              echo "$FILE does not exists"
+            fi
+      - restore_cache:
+
+          keys:
+            - env-v1-linux-{{ arch }}-py<< parameters.python_version >>-{{ checksum ".circleci/unittest/linux/scripts/environment.yml" }}-{{ checksum ".circleci-weekly" }}
+
+      - run:
+          name: Setup
+          command: .circleci/unittest/linux/scripts/setup_env.sh
+      - save_cache:
+
+          key: env-v1-linux-{{ arch }}-py<< parameters.python_version >>-{{ checksum ".circleci/unittest/linux/scripts/environment.yml" }}-{{ checksum ".circleci-weekly" }}
+
+          paths:
+            - conda
+            - env
+      - run:
+          name: Install torchtext
+          command: .circleci/unittest/linux/scripts/install.sh
+      - run:
+          name: Generate daily data Cache
+          no_output_timeout: 30m
+          command: |
+            .circleci/unittest/linux/scripts/generate_cache.sh
+            cat .data/cache_status_file.json
+      - save_cache:
+
+          key: data-linux-v1-{{ checksum ".circleci-daily" }}
+
+          paths:
+            - .data
+      - run:
+          name: Post process
+          command: .circleci/unittest/linux/scripts/post_process.sh
```

Comment:
@parmeet `Post process` is not necessary and it could alter the unittest coverage report so they should be removed.

Reply:
Thanks @mthrok for pointing this out. Let me do a follow-up on this.